liberty-managed/src/main/java/io/openliberty/arquillian/managed/WLPManagedContainer.java

```diff
@@ -202,32 +202,45 @@ public void start() throws LifecycleException
 
 			String javaVmArguments = containerConfiguration.getJavaVmArguments();
 
-            cmd.add(System.getProperty("java.home") + "/bin/java");
-            cmd.add("-Dcom.ibm.ws.logging.console.log.level=INFO");
-
-            // Only add JPMS options for Java 9+
-            if (!System.getProperty("java.specification.version").startsWith("1.")) {
-            		cmd.addAll(getJPMSOptions());
-            }
-            
-            if (!javaVmArguments.equals("")) {
-            	cmd.addAll(parseJvmArgs(javaVmArguments));
-         	}
-            cmd.add("-javaagent:lib/bootstrap-agent.jar");
-            cmd.add("-jar");
-            cmd.add("lib/ws-launch.jar");
-            cmd.add(containerConfiguration.getServerName());
-            
-            log.finer("Starting server with command: " + cmd.toString());
+//            cmd.add(System.getProperty("java.home") + "/bin/java");
+//            cmd.add("-Dcom.ibm.ws.logging.console.log.level=INFO");
+//
+//            // Only add JPMS options for Java 9+
+//            if (!System.getProperty("java.specification.version").startsWith("1.")) {
+//            		cmd.addAll(getJPMSOptions());
+//            }
+//
+//            if (!javaVmArguments.equals("")) {
+//            	cmd.addAll(parseJvmArgs(javaVmArguments));
```

Comment:
The old code would pass along `javaVmArguments` to the server process, and we should preserve this behavior. In 19002 we can just pass them along as arguments in ProcessBuilder, but this plugin still needs to work on older versions of Liberty so we can't rely on that mechanism. For IBM Java and OpenJ9 we can use `IBM_JAVA_OPTIONS` as an environment variable, but we need something that will work for all JDKs. Lets keep digging around and see if we can find the magic option to do this.

Reply:
Ok, after a bit of digging it looks like `JAVA_TOOL_OPTIONS` is the magic env var we want to set on processBuilder